swatch-core/src/main/resources/logback-spring.xml

```diff
@@ -39,4 +39,34 @@
         <appender-ref ref="ConsoleAppender" />
         <appender-ref ref="SplunkAppender" />
     </root>
+
+    <if condition='property("ENABLE_SPLUNK_HEC").equals("true")'>
```

Comment:
Another sort of buried env variable that I would like to be more obvious eventually.